src/libraries/System.Private.CoreLib/src/System/Threading/Thread.cs

```diff
@@ -507,13 +507,13 @@ private bool SetApartmentState(ApartmentState state, bool throwOnError)
             return SetApartmentStateUnchecked(state, throwOnError);
         }
 
-        [Obsolete("Thread.GetCompressedStack is no longer supported. Please use the System.Threading.CompressedStack class")]
+        [Obsolete("Thread.GetCompressedStack has been deprecated. Use the System.Threading.CompressedStack class instead.")]
```

Comment:
Oh, cool. It will be a very minor breaking change to switch the diagnostic ID, but I think it's appropriate here, as any usage of CAS would already need to be accommodated in the build.

I'll make that change as a separate PR so that we have an isolated breaking change.